crates/sui-graphql-rpc/src/test_infra/cluster.rs

```diff
@@ -167,7 +167,7 @@ pub async fn start_graphql_server_with_fn_rpc(
 
     // Starts graphql server
     tokio::spawn(async move {
-        start_graphiql_server(&server_config, &Version("test"))
+        start_graphiql_server(&server_config, &Version(env!("CARGO_PKG_VERSION")))
```

Comment:
Ditto -- make up a test version? Maybe create a constant for the test version? It would in any case help with test stability.

Reply:
This is actually used in the `sui-graphql-e2e-tests` if I am not mistaken, as we have some tests that output the version header.

Reply:
Yeah, I think it's fine to also update those tests to expect the "test" version, because that's really what we're testing here (that the value we describe as the vesion is what is displayed here).